Framework/Algorithms/src/FilterByLogValue.cpp

```diff
@@ -47,20 +47,27 @@ void FilterByLogValue::init() {
   auto min = std::make_shared<BoundedValidator<double>>();
   min->setLower(0.0);
   declareProperty("TimeTolerance", 0.0, min,
-                  "Tolerance, in seconds, for the event times to keep. A good "
-                  "value is 1/2 your measurement interval. \n"
-                  "For a single log value at time T, all events between "
-                  "T+-Tolerance are kept.\n"
+                  "Tolerance, in seconds, for the event times to keep. \n"
```

Comment:
This long of a parameter description, especially with itemized list, is very difficult to read when formatted as a table in the documentation page. The `"\n"` is ignored by the web browser and do not help. It would be better to move much of this information into the main documentation for `FilterByLogValue`.